api/src/resources/projects.py

```diff
@@ -73,3 +77,53 @@ def get(self, id):
 
 
 api.add_resource(ProjectLocationsAPI, '/projects/<int:id>/locations', endpoint='project_locations')
+
+
+class UploadAPI(Resource):  
```

Comment:
Can we change the name of this class to ProjectUploadAPI, since it just provides uploads for the projects resource?